instance.go

```diff
@@ -25,6 +25,18 @@ type instanceResult struct {
 	Result  Result   `xml:"Tasks>Task>Result"`
 }
 
+type instanceErrorMessage struct {
+	CDATA string `xml:",cdata"`
+}
+
+type instanceError struct {
```

Comment:
Is it the general error of instance, or only for the status code 400?

Reply:
You are right. Probably should name it to `requestError`?

Reply:
`requestError` is better.
But I'm worried about if there're several error message structs not involved.
How about we take a look at [aliyun-odps-jdbc](https://github.com/aliyun/aliyun-odps-jdbc) or [aliyun-odps-python-sdk](https://github.com/aliyun/aliyun-odps-python-sdk).

Reply:
Looks like there is only one error struct. :) https://github.com/aliyun/aliyun-odps-java-sdk/blob/master/odps-sdk/odps-sdk-core/src/main/java/com/aliyun/odps/rest/ErrorMessage.java